examples/fromJSON.js

```diff
@@ -33,8 +33,9 @@ const {GoogleAuth} = require('../build/src/index');
  *       "client_id": "your-client-id",
  *       "auth_uri": "https://accounts.google.com/o/oauth2/auth",
  *       "token_uri": "https://accounts.google.com/o/oauth2/token",
- *       "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
- *       "client_x509_cert_url": "your-cert-url"
+ *       "auth_provider_x509_cert_url":
+ *"https://www.googleapis.com/oauth2/v1/certs", "client_x509_cert_url":
```

Comment:
This doesn't look ok to me though. Is there any way to convince clang-format otherwise?

Reply:
Yeah I have no idea what clang-format is trying to do here.  I kind of wonder out loud if it supports the `async` keyword....

Reply:
Doesn't seem related to `async`. The same thing happens without `async` too. See https://github.com/google/google-auth-library-nodejs/pull/233/files#diff-7538725028be787890f7305df2148f61R43